csrankings-h.csv

```diff
@@ -27,6 +27,7 @@ Hadas Shachnai,Technion,http://www.cs.technion.ac.il/~hadas,NOSCHOLARPAGE
 Hadi Amiri,University of Massachusetts Lowell,https://scholar.harvard.edu/hadi,fyUpZ5EAAAAJ
 Hadi Hemmati,University of Manitoba,http://sealab.cs.umanitoba.ca/?page_id=149,TznXpSIAAAAJ
 Hadi Hosseini,Pennsylvania State University,https://faculty.ist.psu.edu/hadi,bnvEtsMAAAAJ
+Hadi Jamali-Rad,TU Delft,https://www.tudelft.nl/en/ewi/over-de-faculteit/afdelingen/intelligent-systems/pattern-recognition-bioinformatics/computer-vision-lab/people/hadi-jamali-rad,l4hw34oAAAAJ

```

Comment:
Web page says he only has a 20% (part-time) appointment. Please remove, per the instructions.

Reply:
I did not notice that, thx! Faculty member was removed.